enterprise/cypher/physical-planning/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/SlotConfiguration.scala

```
@@ -219,6 +219,8 @@ class SlotConfiguration(private val slots: mutable.Map[String, Slot],
           LongSlot(offset, nullable, newSlot.typ)
         case ((RefSlot(offset, nullable, _), false, true)) =>
           RefSlot(offset, nullable, newSlot.typ)
+        case _ =>
+          existingSlot // This should never happen but it eliminates a warning
```

If it should never happen, it fells better to throw an exception instead of non-failing and silently at that.